client/rpc/src/system/tests.rs

```diff
@@ -104,6 +104,9 @@ fn api<T: Into<Option<Status>>>(sync: T) -> System<Block> {
 						Err(s) => sender.send(Err(error::Error::MalformattedPeerArg(s.to_string()))),
 					};
 				}
+				Request::NetworkReservedPeers(sender) => {
+					let _ = sender.send(vec!["peer_A".to_string()]);
```

Comment:
Would it be possible to use something realistic instead? I believe it should be a `PeerId`, like `12D...`

Reply:
Will do